tools/scripts/unit_test_coverage.sh

```diff
@@ -25,7 +25,17 @@ function get-units {
             sed -E -e '/[CS]\.(h|cpp|inl)$/ d' -e '/TypeSupportImpl\.(h|cpp)$/ d' -e '/[Ee]xport.h$/ d' -e 's/\.[^.]*$//' |
             sort -u
     else
-        echo "${args[@]}"
+        for arg in "${args[@]}"
+        do
+            if [[ -d "${arg}" ]]
+            then
+                find "${arg}" -path tools/rapidjson -prune -false -o -path tools/modeling/tests -prune -false -o -name "*.h" -o -name "*.cpp" -o -name "*.inl" |
+            sed -E -e '/[CS]\.(h|cpp|inl)$/ d' -e '/TypeSupportImpl\.(h|cpp)$/ d' -e '/[Ee]xport.h$/ d' -e 's/\.[^.]*$//' |
+            sort -u
```

Comment:
Indent level looks off - but not sure what it should be for shell.

Pattern matching on file names to determine generated files may not be accurate.  Alternatives to consider would be running a git command (assuming this is always a repo), or having another mode of the script that would run before `make` and create a text file listing the non-generated source file names.

Reply:
I am using `git` to check for ignored files in the [lint script](https://github.com/objectcomputing/OpenDDS/blob/05c82664fc481a2d9cc620d3eb4f00f9911c14ff/tools/scripts/lint.pl#L1059-L1072).